experimental/rule_inference/base_prompt.py

```diff
@@ -105,7 +105,7 @@ class BasePrompt:
 
     @staticmethod
     def generate_prompt(**kwargs):
-        examples = BasePrompt._get_examples("../../src/cleanup_rules")
+        examples = BasePrompt._get_examples("../../src/cleanup_rules/java")
```

Comment:
Unrelated to this jave. But this is to prevent the prompt from becoming too large :|